src/js/common/schemaform/validation.js

```diff
@@ -255,6 +255,12 @@ export function validateFutureDateIfExpectedGrad(errors, dateString, formData) {
   }
 }
 
+export function validateYear(errors, year) {
```

Comment:
The logic here is a little different as it takes a year rather than a full date, also it checks whether the year is between 1900 and the current year. I wonder if making changes to `isValidCurrentOrPastYear` to accommodate these differences is what you had in mind? Or is simply checking that a year is in the past sufficient and the after 1900 check unnecessary?